bin/yarn

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
```

Comment:
Whats this file about?

Reply:
This is an executable file which allows us to run the yarn-cli that has commands like `yarn ...`. Yarn is a package manager, an alternative to npm.

Reply:
We all know that if we install yarn globally (eg: `npm i -g yarn`), we have access to the yarn command. If thats the case, why this?

Reply:
This should also allow someone else who has not installed yarn globally to execute the `yarn` commands so long as they are in the project directory.